node/node.go

```diff
@@ -2020,7 +2020,7 @@ func (app *App) startSynchronous(ctx context.Context) (err error) {
 	app.errCh = make(chan error, 100)
 	if app.Config.PprofHTTPServer {
 		logger.Info("starting pprof server")
-		app.pprofService = &http.Server{Addr: ":6060"}
+		app.pprofService = &http.Server{Addr: "127.0.0.1:6060"}
```

Comment:
it is better to expose option to change the address, otherwise it may end up being too restrictive and will require workarounds (such as tunneling) when it would be totally fine to expose it on some private ip or even 0.0.0.0

Reply:
Ok. Makes sense. Will do.